common/reactive/src/test/java/io/helidon/common/reactive/SequentialSubscriberTest.java

```diff
@@ -98,7 +99,7 @@ public void onComplete() {
         } catch (InterruptedException e) {
             executorService.shutdownNow();
         }
-        assertFalse(errorFound.isPresent(), () -> errorFound.get());
+        assertThat(errorFound.orElse(""), errorFound.isPresent(), is(false));
```

Comment:
I don't like this approach with `Optional`, but if you need the error message...

Reply:
For Helidon 4 (`main` branch), we do have an optional matcher. For this version, we can keep this.

Alternative:
```java
assertThat(errorFound, is(Optional.empty()));
```